actions/lib/helpers.py

```diff
@@ -45,13 +45,17 @@ def get_issue_url(api_client, issue_id, portal_url):
     return url
 
 
-def convert_timestamps_to_str(dictionary: dict) -> dict:
-    for k, v in dictionary.items():
-        try:
-            dictionary[k] = v.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
-        except AttributeError:
-            pass
-    return dictionary
+def convert_timestamps_to_str(json_element):
```

Comment:
This function deserves a good doc-string explaining what it does with `json_element` (it does change it).

Also, a json_element can be a string, that is not covered here (it will also not work).

If string is not wanted, you should really use typing:

```convert_timestamps_to_str(json_element: Union[dict, list]) -> NoReturn```